### ISSUE TITLE
 quotes in comments break stylelint in less mode

### ISSUE THREAD

**Issue by Zae** [shellscape/postcss-less#145]
Hi,

I noticed that stylelint does not handle single quotes in // comments well. 

This might be a duplicate of #130 but i'm still having this issue.

I can sort of reproduce it in the demo environment, but not really. Locally the quote causes the linenumbers to start at 0 at the quote again, so errors following the quote will have incorrect linenumbers.

In the demo the duplicate selector isn't found by stylelint, but only if the quote if on the end of the line, locally it doesn't matter where the quote is in the comment.

/* */ comments work fine btw.

Version: 9.10.1 and 11.1.1 (did not test any other versions)

@evilebottnawi asked me to open an issue (https://github.com/stylelint/stylelint/issues/4411) here as he thinks the issue is caused by postcss-less.

* Node Version: v10.15.0 (but i don't think it matters)
* NPM Version: 6.4.1 (but i don't think it matters)
* YARN Version: 1.19.1 (but i don't think it matters)
* postcss Version: ^7.0.14
* postcss-less Version: ^3.1.4

### LESS

```less
a {
  color: #FFF;
}

p {
  color: #ASDASD;
  // Dont do this it breaks'
}

a {
  color:#BROKEN;
}
```

### Expected Behavior
Stylelint shows the correct line

### Actual Behavior
Stylelint resets the linenumbers when a quote is found in a single line comment.

### How can we reproduce the behavior?
Try to stylelint the provided less

**Comment by shellscape** [comment]
Thanks for opening an issue :beer:

I would be happy to review a pull request which addressed the line numbering problem.

**Comment by popov-tensor** [comment]
There is an "extended version" of bug described in https://github.com/stylelint/stylelint/issues/5115 .
If there is a quote character in lines below the problem comment, stylelint still fails to number lines correctly. The test above contains only one quote character in comment, so does not cover such case.

**Comment by shellscape** [comment]
Fixed by #154
